src/Uno.UI/UI/Xaml/Controls/Grid/ColumnDefinition.cs

```diff
@@ -42,5 +42,41 @@ private void OnWidthChanged(DependencyPropertyChangedEventArgs e)
 		{
 			return new ColumnDefinition { Width = GridLength.ParseGridLength(value).First() };
 		}
+		
+		public double MinWidth
+		{
+			get
+			{
+				return (double)this.GetValue(MinWidthProperty);
+			}
+			set
+			{
+				this.SetValue(MinWidthProperty, value);
+			}
+		}
+
+		public double MaxWidth
+		{
+			get
+			{
+				return (double)this.GetValue(MaxWidthProperty);
+			}
+			set
+			{
+				this.SetValue(MaxWidthProperty, value);
+			}
+		}
+
+		public static DependencyProperty MinWidthProperty { get; } =
+		DependencyProperty.Register(
+			"MinWidth", typeof(double),
+			typeof(ColumnDefinition),
+			new FrameworkPropertyMetadata(default(double)));
```

Comment:
Explicit `0.0` instead ?

Reply:
That would be `0.0d`
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/value-types-table

Reply:
will do